app/jobs/update_jobs/adjust_question_variable_references.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class UpdateJobs::AdjustQuestionVariableReferences < UpdateJobs::VariableReferencesUpdateJob
+  def perform(question_id, old_variable_name, new_variable_name)
+    return if old_variable_name == new_variable_name
+    return if old_variable_name.blank? || new_variable_name.blank?
+
+    question = Question.find(question_id)
```

Comment:
Do we want to reschedule the job when no question is found? This is the default behavior for exceptions.